src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -821,6 +821,16 @@ private Bridge selectBridge(Participant participant)
         return bridge;
     }
 
+    /**
+     * Get a stream of those bridges which are operational.
+     * Caller should be synchronized on bridges.
+     */
+    private Stream<BridgeSession> operationalBridges()
```

Comment:
Can we synchronize access to `bridges` to prevent new code from calling the function without synchronizing?

Reply:
No, because we return a `Stream<>` that iterates over `bridges`.